File: HoWo7/ShopService/Sell.py
from Phone.Phone import Phone
from ShopService.PhoneMag import PhoneMag


class Sell:
    @staticmethod
    def changingTheBalance(other_phone:Phone, mag:PhoneMag):
        priceList = mag.getPriceList()
        for phone in priceList:
            if phone == other_phone:
                priceList.remove(phone)
                mag.setPriceList(priceList)
                mag.setMoney(mag.getMoney()+other_phone.getPrice())
                break;

File: HoWo5_6/ShopService/Sell.java
package HoWo5_6.ShopService;

import java.util.ArrayList;

import HoWo5_6.Phone.Phone;

public class Sell implements IntrfTransaction{

    @Override
    public void changingTheBalance(Phone phone, PhoneMag mag) {
        ArrayList<Phone> priceList = mag.getPriceList();
        for (Phone phoneL : priceList) {
            if (phoneL.equals(phone)){
                priceList.remove(phoneL);
                mag.setPriceList(priceList);
                mag.setMoney(mag.getMoney()+phone.getPrice());
                break;
            }
        }
    }
}
